apps/web/src/components/Menu/AppsMenu/AppsMenu.tsx

```diff
@@ -2,21 +2,29 @@ import { Button, Divider, Text } from "@chakra-ui/react";
 import { useAddPeer } from "@umami/state";
 
 import { BeaconPeers } from "../../beacon";
+import { useOnWalletConnect } from "../../WalletConnect";
 import { DrawerContentWrapper } from "../DrawerContentWrapper";
 
 export const AppsMenu = () => {
   const addPeer = useAddPeer();
```

Comment:
I think it would be better to rename this function to something like `useOnBeaconConnect` to make it clearer what each function is responsible for